R/structural.properties.R

```diff
@@ -293,12 +293,13 @@ graph.knn <- function(graph, vids = V(graph), mode = c("all", "out", "in", "tota
 #'
 #' `graph.dfs()` was renamed to `dfs()` to create a more
 #' consistent API.
+#' @param father Logical scalar, whether to return the father of the vertices.
```

Comment:
we can't inherit the function definition anymore as we are not adding the new argument to the old function. 🤪